protobuf/module_msg.proto

```diff
@@ -301,6 +301,16 @@ message WildcardMatchCommandSetDefaultGateArg {
 }
 
 /**
+ * The module WildcardMatch has a function `get_rules()` which
+ * returns the current set of running rules.
+ */
+message WildcardMatchCommandGetRulesResponse {
+  uint64 default_gate = 1; // The default gate
+  WildcardMatchArg fields = 2; // The fields provided in initializing WCM
+  repeated WildcardMatchCommandAddArg rules = 3; //All rules provided via calls to `WilcardMatch.add(...)`
```

Comment:
The name `WildcardMatchCommandAddArg` is a little awkward to be used here. How about `WildcardMatchRule`?